v2/component_analyses.yaml

```diff
@@ -0,0 +1,140 @@
+security:
+- tokenAuth: []
+paths:
+  /component-analyses/{eco}/{name}/{ver}:
+    get:
+      tags:
+      - Scan Services
+      summary: Get security and other details about a software component and recommendations
+        to change to a more secured version
+      operationId: f8a_scanner.api_v2.component-analyses
+      parameters:
+      - name: eco
+        in: path
+        description: component ecosystem
+        required: true
+        schema:
+          type: string
+      - name: name
+        in: path
+        description: component name
+        required: true
+        schema:
+          type: string
+      - name: ver
+        in: path
+        description: component version
+        required: true
+        schema:
+          type: string
+      - name: security-vendor
+        in: header
+        description: Security Vendor
+        schema:
+          type: string
+          enum:
+          - snyk
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ComponentAnalysesResponse'
+        202:
+          description: Package is currently Unavailable. Please try again later
+          content: {}
+        401:
+          description: Request unauthorized
+          content: {}
+        404:
+          description: Data not found
+          content: {}
+components:
+  schemas:
+    VulnerabilityDetails:
+      title: Vulnerability Details
+      type: object
+      properties:
+        vulneability:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+              cvss:
+                type: string
+      description: Information of the vulnerability found
+
+    PrivateRecommendation:
+      title: Vendor Private Recommendation Information
+      type: object
+      properties:
+        change_to:
+          type: string
+        component-analyses:
+          $ref: '#/components/schemas/VulnerabilityDetails'
+        message:
+          type: string
+        severity:
+          type: string
+        public_vulnerabilities_count:
+          type: integer
+        private_vulnerabilities_count:
+          type: integer
+        exploitable_vulnerabilities_count:
+          type: integer
+        vendor_package_link:
+          type: string
+      description: Vendor Recommendation information for Premium Recommendation.
+
+    PublicRecommendation:
+      title: Vendor Public Recommendation Information
+      type: object
+      properties:
+        change_to:
+          type: string
+        component-analyses:
+          $ref: '#/components/schemas/VulnerabilityDetails'
+        message:
+          type: string
+        severity:
+          type: string
+        public_vulnerabilities_count:
+          type: integer
+        vendor_package_link:
```

Comment:
Can you change to url? or more meaningful one?

Reply:
IMHO `vendor_package_link` is more self-explanatory than plain `url`.